taipy/gui/_renderers/factory.py

```diff
@@ -597,6 +598,20 @@ class _Factory:
             ]
         )
         ._set_propagate(),
+        "Progress": lambda gui, control_type, attrs: _Builder(
+            gui=gui,
+            control_type=control_type,
+            element_name="Progress",
+            attributes=attrs,
+        )
+        .set_value_and_default(var_type=PropertyType.dynamic_number, with_default=False)
+        .set_attributes(
+            [
+                ("linear", PropertyType.boolean, False),
+                ("showProgress", PropertyType.boolean, False),
```

Comment:
property names have to be aligned (and in a pythonic way)
`showProgress `=> `show_value`

Reply:
Oh! I forgot to rename it in `factory.py`, will do that👍